client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx

```diff
@@ -94,30 +151,32 @@ const MarketplaceThankYou = ( { productSlug } ): JSX.Element => {
 	};
 
 	const thankYouSubtitle = translate( '%(pluginName)s has been installed.', {
-		args: { pluginName: plugin?.name },
+		args: { pluginName: pluginOnSite?.name },
 	} );
 
 	return (
 		<ThemeProvider theme={ theme }>
-			<Masterbar>
-				<Item
-					icon="cross"
+			<MasterbarStyled>
+				<WordPressLogoStyled />
+				<ItemStyled
+					icon="chevron-left"
 					onClick={ () =>
 						( document.location.href = `${ document.location.origin }/plugins/${ siteSlug }` )
 					} // Force reload the page.
-					tooltip={ translate( 'Go to home' ) }
-					tipTarget="close"
-				/>
-				<WordPressWordmarkStyled />
-			</Masterbar>
+				>
+					{ translate( 'Back to marketplace' ) }
```

Comment:
I don't think we are using the *marketplace* term anywhere other than URLs. The page title shows *Plugins* so we may want to stick with it here.

```suggestion
					{ translate( 'Back to Plugins' ) }
```

Reply:
Good point da21e63